src/decoder/Bridge.cxx

```diff
@@ -407,13 +407,18 @@ try {
 
 	std::lock_guard<Mutex> lock(is.mutex);
 
+	int try_cnt = 5;
 	while (true) {
 		if (CheckCancelRead())
 			return 0;
 
 		if (is.IsAvailable())
 			break;
 
+		if (--try_cnt <= 0) { // walkaround for some curl input no respond.
```

Comment:
Bad workaround. What was the problem? Let's fix this problem properly.

Reply:
Seem to be playing internet music, then the router is disconnected from the internet. MPD may be blocked at this loop. I can't recall, sorry.